templates/pastes.html

```diff
@@ -0,0 +1,74 @@
+{% extends 'base.html' %}
+{% block head %}
+  {% load static %}
```

Comment:
It's safe to add a {% load %} outside of a {% block %}, because its work is done at parse time, not render time.